contracts/SnapshotToken/Helpers/Allowance.sol

```diff
@@ -80,6 +80,9 @@ contract Allowance is
         public
         returns (bool success)
     {
+        // AUDIT[CHF-116] Unnecessary require() in Allowance.approveAndCall()
```

Comment:
This is why we dropped all `false` return value forwarding/bubbling up in all of our tokens. Whenever there is `false` we immediately revert. Already in master branch... 
However I didn't think about that case. Good catch. We base our implementation on MiniMe token. I'll let them know about the problem.